etc/systemd/system/zfs-import-scan.service.in

```diff
@@ -6,6 +6,7 @@ Requires=systemd-udev-settle.service
 After=systemd-udev-settle.service
 After=cryptsetup.target
 Before=dracut-mount.service
+Before=zfs-import.target
```

Comment:
AFAICT, the "WantedBy" below already implies a "Before", so this is probably redundant?

see "man systemd.unit":
```
       A number of unit dependencies are automatically established, depending on unit configuration. On top of that, for units with
       DefaultDependencies=yes (the default) a couple of additional dependencies are added. The precise effect of DefaultDependencies=yes
       depends on the unit type (see below).

       If DefaultDependencies=yes is set, units that are referenced by other units of type .target via a Wants= or Requires= dependency
       might automatically gain an Before= dependency too. See systemd.target(5) for details.
```

Reply:
I don't think that a `Wants=` dependency implies an `After=` dependency (also `man systemd.unit`, under `Requires=`

```
           dependencies for all listed names will be created. Note that
           requirement dependencies do not influence the order in which services
           are started or stopped. This has to be configured independently with
           the After= or Before= options. If a unit foo.service requires a unit
           bar.service as configured with Requires= and no ordering is
           configured with After= or Before=, then both units will be started
           simultaneously and without any delay between them if foo.service is
           activated. Often, it is a better choice to use Wants= instead of
           Requires= in order to achieve a system that is more robust when
           dealing with failing services.
```

and

```
    Wants=
           A weaker version of Requires=. Units listed in this option will be
```

Reply:
yes, sorry I missed it. still not really used to the GH diff view..